usefulUtilities/loudApp/appResources/loudApp_app.js

```diff
@@ -0,0 +1,559 @@
+
+            if (hasUUID && !isInNewList) {
+                removeUser(uuid);
+            }
+        }
+        doUIUpdate();
+    }
+
+
+    // Delete from top ten loudest list
+    function removeUserFromSettings(uuid) {
+        if (selectedUserUUID === uuid) { // targeted user with yellow overlay
+            deselectUserOverlay(uuid);
+        }
+        if (activeTargetUUID === uuid) { // is listening to this user
+             stopListening();
+        }
+        var settingsUsersListIndex = getIndexOfSettingsUser(uuid);
+        if (settingsUsersListIndex !== -1) {
+            if (settings.users[settingsUsersListIndex].overlayID) {
+                deleteOverlay(uuid);
+            }
+            settings.users.splice(settingsUsersListIndex, 1);
+            doUIUpdate();
+        }
+    }
+
+    // #endregion UPDATE
+
+
+    // #region WEBEVENTS
+
+    // Handles messages from UI
+    var LISTEN_TOGGLE = "listen_toggle",
+        SELECT_AVATAR = "selectAvatar",
+        BAN = "ban",
+        MUTE = "mute",
+        REFRESH = "refresh",
+        GOTO = "goto",
+        UPDATE_UI = "update_ui",
+        SEARCH_WHOLE_DOMAIN_FOR_LOUDEST_ENABLED = "searchWholeDomainForLoudestEnabled",
+        TOGGLE_EXPANDING_AUDIO = "toggleExpandingAudio";
+    function onMessage(data) {
+        if (DEBUG) {
+            console.log("LoudApp recieved message: " + data.type);
+        }
+
+        switch (data.type) {
+            case EVENT_BRIDGE_OPEN_MESSAGE:
+                if (!settings.users.length) {
+                    // only add people to the list if there are none
+                    sortAvatarsByLoudness();
+                }
+                doUIUpdate();
+                break;
+
+            case LISTEN_TOGGLE:
+                var avatarInfo = data.value;
+                if (DEBUG) {
+                    console.log("LISTEN TOGGLE ", avatarInfo.uuid !== activeTargetUUID, JSON.stringify(avatarInfo));
+                }
+                if (avatarInfo.uuid !== activeTargetUUID) {
+                    startListeningToAvatar(avatarInfo.uuid);
+                } else {
+                    stopListening();
+                }
+                break;
+
+            case SELECT_AVATAR:
+                var avatarInfo = data.value;
+                var orientationTowardsUser = Quat.cancelOutRollAndPitch(Quat.lookAtSimple(MyAvatar.position, avatarInfo.currentPosition));
+                MyAvatar.orientation = orientationTowardsUser;
+                if (selectedUserUUID === avatarInfo.uuid) {
+                    deselectUserOverlay(avatarInfo.uuid);
+                } else {
+                    selectUserOverlay(avatarInfo.uuid);
+                }
+                break;
+
+            case REFRESH:
+                sortAvatarsByLoudness();
+                doUIUpdate();
+                stopListening();    
+                break;
+
+            case GOTO:
+                // teleport behind the target avatar
```

Comment:
https://knowyourmeme.com/memes/teleports-behind-you-nothing-personal-kid

Reply:
You should see someone in a small avatar like salina use "goto" behind everyone.

Reply:
:P